src/components/views/causes/CausesSearchDesktop.tsx

```
@@ -0,0 +1,113 @@
+			query: updatedQuery,
+		});
+	};
+
+	const [inputRef, setFocus] = useFocus();
+
+	const removeSearch = () => {
+		setSearchValue('');
+		const updatedQuery = {
+			...router.query,
+		};
+		delete updatedQuery.searchTerm;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)